APIURLS.md

```diff
@@ -0,0 +1,571 @@
+		   "type": "Team-managed business",
+		  “lead”: ObjectId("UserID")
+                }
+	     Response:
+                {
+                    "success": true,
+                    "msg": "Project created successfully.",
+                    "project": {
+	      		  "id": ObjectId,
+			  "name": "Project Name",
+			   "key": "Project Initials",
+			   "type": "Team-managed business",
+			  “lead”: ObjectId("UserID")
+                    }
+                }
+	Project setting:
+ 	    PUT loclalhost:8080/API/v1/projects?id=x&name=y&key=a&type=b&lead=c
+      	Last Issue Update:
+       	    PATCH loclalhost:8080/API/v1/projects/id/lastIssueUpdates
+        Search Project:
+            GET localhost:8080/API/v1/projects?id
+	    response:
+     		{
+                    "success": true,
+                    "projects": {
+	      		  "id": ObjectId,
+			  "name": "Project Name",
+			  "key": "Project Initials",
+			  "type": "Team-managed business",
+			  "lead": ObjectId("UserID")
+        	}
+        Delete Project:
+            Delete localhost:8080/API/v1/projects?id
+	    Request:
+		DELETE /API/v1/projects?id=项目ID HTTP/1.1
+		Host: localhost:8080
+		Content-Type: application/json
+		Authorization: Bearer your_access_token
+  	    Response:
+       		{
+		    "status": "success",
+		    "msg": "Project deleted successfully"
+		}
+        List Project Table:
+            GET localhost:8080/API/v1/projects
+	    Response:
+     		              {
+                    "success": true,
+                    "projects": [
+                        {
+                            "id": "101",
+                            "name": "exampleProject1",
+			    "key": "Project Initials",
+			    "type": "Team-managed business",
+			    "lead": ObjectId("UserID")
+                        },
+                        {
+                            "id": "102",
+                            "name": "AI for Healthcare",
+		  	    "key": "Project Initials",
+			    "type": "Team-managed business",
+			    "lead": ObjectId("UserID")
+                        }
+                    ]
+                }
+    Product Access:
+    	Product Database:
+     	    localhost:8080/API/v1/access
+            [Post, Put, Get, Delete]
+ 	PUT localhost:8080/API/v1/accesses?User_id=1&useradmin=2&productadmin=3
+    User Management:
+        Users Database:
+            localhost:8080/API/v1/User/id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/Users
+        Invite User: (send user invitation by email)
+            POST localhost:8080/API/v1/boards/id/members
+	    Request:
+     		{
+		  "email": "example@ex.com"
+		}
+     	    Response:
+	  	{
+		  "status": "success",
+		  "message": "User invited successfully to the board."
+		}
+  		or
+    		{
+		  "status": "error",
+		  "message": "User with the provided email not found."
+		}
+        List User Table:
+            GET localhost:8080/API/v1/boards/board_Id/memebers

```

Comment:
Same comments:
Wrong URL: user and board are independent, only that they are associate.